response_operations_ui/controllers/mock_party_controller.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+from flask import current_app as app
+
```

Comment:
So, for clarity, this is the file that replaces `party_controller.py` to allow UI-only tests to run without spinning up party or other dependencies.

The long term plan is that `party_controller` will become a class, and mock data will be handled in that, in a similar way to how party does it.  I'm adding a ticket to the backlog for that.